IdentityCore/src/MSIDConstants.h

```diff
@@ -81,10 +83,20 @@ typedef NS_ENUM(NSInteger, MSIDAuthScheme)
     MSIDAuthSchemePop,
 };
 
+typedef NS_ENUM(NSInteger, MSIDGetHeaderType)
+{
+    MSIDGetAllHeader = 0,
```

Comment:
Can we follow the same naming convention as in other enums? Also, I don't think we need "get" preffix. Suggest to change to something like this:
```
typedef NS_ENUM(NSInteger, MSIDHeaderType)
{
   MSIDHeaderTypeAll,
   MSIDHeaderTypePrt,
   MSIDHeaderTypeDeviceRegistration
}
```

Reply:
Updated